content/post/stats_at_work.md

```diff
@@ -7,89 +7,72 @@ date: 2018-07-26T20:35:46-04:00
-MA: Kind of like how a confidence interval around our estimate can be wide if our sample is small? I'm assuming if the sample is large our confident interval is narrow? 
+Diane: Well, that varies. You're right, we set our significance threshold to 5%. We also set our desired effect size, aka the minimum detectable difference, to be 0.5%. So the question is simply: what is the minimum number of samples we need to detect that difference. That's our sample size. And in the formula, it turns out that the power level associated with that sample size is roughly 80%. Of course, if we collected more samples, our power would increase. But we're often content with 80%.  
+Marty: What? 80%? Why are we happy with that?
 
-DS: Precisely.  
-MA: Cool. Power and sample size and confidence in our estimate are all related. But what sample size are we aiming for? Like what's the "5% significance level" equivalent for sample size?
+Diane: Honestly, it's sort of an artifact of how things have always been done. Back in the day some smart people figured that making a false positive error was 4 times worse than making a false negative error. So if we agree that a 5% significance is a good false positive rate to protect against, then 20% is a good false negative rate. And 1 minus this number, which is 80%, is our power.   
+Marty: Oh, so it's kind of made up.
 
-DS: Well, that varies. You're right, we set our significance threshold to 5%. We also set our desired effect size to 0.5%. Now the question is - how many data points do we need to be confident that our conclusion can be trusted. But let me ask you - how do we quantify "trust level"?  
-MA: I don't know. 95%?
+Diane: Yeah. So is the 5% significance level. But it works surprisingly well in helping us make decisions. And all that allows us to stop the test when we do.  
+Marty: I see. So before we start the test, we set our significance threshold, the desired effect we're looking for, and some sample size number. This then guarantees that when we hit that sample size and stop our test, our p-value is...trustworthy? 
 
-DS: Yeah that's tempting cause we chose 95% for the confidence level (which results in a 5% significance threshold level) to determine if the p-value is significant or not. Let's flip the "trust level" question. Let's consider what our trust level would be with 10 users.  
-MA: Hmm...probably small.
+Diane: Right. It basically comes down to the p-value settling down after the sample is large enough. Early on in the test the p-value calculation jumps around too much, and you're more likely to find a p-value under 5% and think that there is a difference, even though it's not really trustworthy yet. You need to wait until the number stops jumping around so much. It sort of converges.  
```

Comment:
I think there's a bit more nuance here that you're eliding (perhaps strategically for clarity). The issue is not necessarily that the p-value jumps around, but that in practice the "stopping rule" is asymmetric ... people will only stop the test early if the thing they want to win is winning, but they'll "let it run a little longer" if the p-value is > 0.05.

Happy to leave this out as I think for the target audience of the post it's pretty far into the weeds...

Reply:
I can add:

```
Early on in the test the p-value calculation jumps around too much, and since you're sort of looking for a significant result, you're more likely to stop the test when the p-value happens to drop under 5% and think that there is a difference, even though it's not really trustworthy yet.
```